README.md

```diff
@@ -61,11 +74,108 @@ Rails.application.config.emergency_banner_redis_client = Redis.new(
 )
 ```
 
+## Adding global banners
+
+Global banners are passed to the [Layout for
+Public](https://components.publishing.service.gov.uk/component-guide/layout_for_public)
+component, which is currently applied to each frontend app by the slimmer/static
+wrapping code - so you will only need to handle global banners in your app when
+Slimmer is removed from it. Once Slimmer is removed and you are calling the
+layout_for_public component directly in your app, add the global banner partial
+to the component's `global_bar:` key:
+
+```
+<%= render "govuk_publishing_components/components/layout_for_public", {
+  draft_watermark: draft_environment,
+  global_bar: render("govuk_web_banners/globla_bar"), # <-- Add this line
+  full_width: false,
+  ...etc
+```
+
+## Updating banner information in the gem
+
+Data for the global banners can be found at
+`config/govuk_web_banners/global_banners.yml`. To add a banner to the config,
+add an entry under the banners: array. Note that this array must always be
+valid, so if there are no banners in the file, it must contain at least
+`global_banners: []`
+
+### Example banner entry
+
+```
+global_banners:
+- name: Banner 1
+  items:
+  - title: "Register to Vote"
+    title_href: /register-to-vote
+    text: "You must register to vote before the election"
+  show_arrows: false
+  permanent: false
+  exclude_paths:
+  - /find-your-local-electoral-office
+  start_date: 21/10/2024
+  end_date: 18/11/2024
+```
+Unlike the other banners, a global banner can contain more than one item, so
+each banner record must contain an `items` array with one or more items, each
+of which must include a `title`, `title_href`, `text`, and the banner as a whole
+must include a valid `start_date`. `title_href` can be either a valid URL or a
+path on gov.uk. (NB: multiple items not supported in the actual component yet,
+that will be added later)
+
+> [!NOTE]
+>
+> `start_date` is **mandatory** here (unlike in recruitment banners) because it's
+> needed to create a banner_version to pass to the underlying component. This lets
+> the component reset the cookie that records how many times a banner has been seen
+> (by default banners are shown only three times, see the `permanent` option below.)
+
+Optional keys are:
+- `name` (an identifying name for this banner, not rendered
+  anywhere)
+- `show_arrows` (defaults to false, if true shows arrows before each
```

Comment:
Just interested... are these arrows going to be shown in the banner? Is this referring to action links?

Reply:
Designers would like the arrows, but I think they're somewhat tied into the "two banners at the same time" situation, which we're deferring for the moment, so I've removed this.

Reply:
Designers would like the arrows, but I think they're somewhat tied into the "two banners at the same time" situation, which we're deferring for the moment, so I've removed this.